src/redux/actions/sync.js

```diff
@@ -228,3 +230,30 @@ export function doResetSync() {
       resolve();
     });
 }
+
+export function doSyncEncryptAndDecrypt(oldPassword, newPassword, encrypt) {
```

Comment:
When would encrypt be true/false?

Reply:
We are passing it from the desktop side so we know if we are decrypting or encrypting. Wanted to re-use the same function since most of the processing is the same.  If not, we can create 2 separate functions.